coc/clans.py

```diff
@@ -200,17 +208,26 @@ def _from_data(self, data: dict) -> None:
             member_cls(data=mdata, client=self._client, clan=self) for mdata in data_get("memberList", [])
         )
 
+        capital_district_cls = self.capital_district_cls
+        self._iter_capital_districts = (capital_district_cls(data=cddata, client=self._client) for cddata in
+                                        data_get("clanCapital")["districts"])
```

Comment:
Do we know what happens if a clan hasn't started (or unlocked) clan capital yet? (Ie a lv1 clan) - is the clanCapital field still there? This will break otherwise 🙂

Reply:
Yes you are right. Fixed it with the next commit